src/linter/ui5Types/SourceFileLinter.ts

```diff
@@ -204,7 +204,8 @@ export default class SourceFileLinter {
 				if (ts.isPropertyAssignment(prop) &&
 					(ts.isIdentifier(prop.name) || ts.isStringLiteralLike(prop.name)) &&
 					ts.isStringLiteralLike(prop.initializer) &&
-					["definition", "fragmentContent", "viewContent"].includes(prop.name.text)) {
+					["definition", "fragmentContent", "viewContent"].includes(prop.name.text) &&
```

Comment:
The same check is done again below in `isXmlInJsCreationCall` isn't it?
I wonder whether it would be easier to combine `isXmlInJsCreationCall` and `getViewTypeFromXMLInJsCreationCall` as they are very similar. Also, as a CallExpression is required, wouldn't it be better to use the CallExpression as indication node to start the checking instead of the isObjectLiteralExpression inside?

Reply:
I would expect that this fits into the same category as the partial deprecations, where we perform checks for a specific function call of a module / class.